rdr_service/model/bq_questionnaires.py

```diff
@@ -181,7 +182,7 @@ def get_fields(self):
                 # flag duplicate fields.
                 found = False
                 for fld in fields:
-                    if fld['name'] == name:
+                    if fld['name'].lower() == name.lower():
```

Comment:
It looks like there might be a duplicate of this in the resource generators (https://github.com/all-of-us/raw-data-repository/blob/devel/rdr_service/resource/schemas/questionnaires.py#L160). Should it be updated there too?

Reply:
Yes, I just noticed that too -- thanks!  Fixing